recipes/use-reasonml/craco.config.js

```diff
@@ -0,0 +1,12 @@
+// Offical documentation available at: https://github.com/FormAPI/craco-preact
+const { ESLINT_MODES } = require("@craco/craco");
+
+module.exports = {
+  eslint: {
+    mode: ESLINT_MODES.file,
+    // Add `"eslintIgnore":["./src/**/*.js"],` to your `package.json`
+    loaderOptions: eslintOptions => {
+      return { ...eslintOptions, ignore: true };
+    }
+  }
```

Comment:
Does your goal is to turn off eslint?

If so, you can do the following:

```
module.exports = {
    eslint: {
       enable: false
    }
};
```

Reply:
The goal is to turn off eslint for the bucklescript generated files in the src directory. I want to enable that ignoring certain file patterns. I guess I want to disable this https://github.com/facebook/create-react-app/blob/e50e795429264477d3b7181f1d7b590295a7b9e6/packages/react-scripts/config/webpack.config.js#L329

Reply:
Got it! That being said, I am not sure that configuring eslint to ignore a folder qualify for being a recipe. Is there anything else to setup with CRACO for reasonml?

Reply:
Guess it been annoying me for a while and came across your package in some cra docs...and it worked. Wanted to share the love. But not necessary is not necessary.